package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Creates user interface and display for products",
   "scripts": {
     "build": "webpack --config webpack.config.js",
-    "build:watch": "webpack --watch --config webpack.config.js",
+    "watch": "webpack --watch --config webpack.config.js",
```

Comment:
`build:watch` retains the semantic relationship between this script and `build`.
if you want to change it use something like `build:w` or change the `build` script to `build:once`